README.md

```diff
@@ -22,44 +22,58 @@ For each item, we position it in one of four states:
 
 The radar only applies to all actively developed products at Redgate. 
 
-Items in the [primary Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmaster%2Fradar.csv) have an expectation that it should be followed (where appropriate) for all teams at Redgate. For example, if C# X arrives then we get most benefit from adopting it wholesale across the organization, whereas a particular date/time library is a local choice for a development team. The  [Library Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmaster%2Fradar_libraries.csv) aims to support teams with local decisions by sharing experience from other teams.
+Items in the [primary Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmain%2Fradar.csv) have an expectation that it should be followed (where appropriate) for all teams at Redgate. For example, if C# X arrives then we get most benefit from adopting it wholesale across the organization, whereas a particular date/time library is a local choice for a development team.
+
+## Libraries
+
+The [Library Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmain%2Fradar_libraries.csv) aims to support teams with local decisions by sharing experience from other teams.
+
+Libraries are placed into one of four categories:
+* Npm packages
+* Public NuGet packages
+* Redgate NuGet packages
+* Automated Testing
 
 ## Who builds the radar?
 
 The radar is open for anyone within Redgate to contribute.
 
+Before contributing please read the [Contributing Guidelines](.github/CONTRIBUTING.md).
+
 The recommended way to propose a change or spark a discussion is to open a PR.
 
-Contributions are reviewed by Redgate’s Technical Advisory Board (TAB). The TAB consists of a group of senior technology leaders within Redgate (Lead Software Engineers, Head of IT, Head of Product Engineering) and is led by the CTO.
+Contributions are reviewed by Redgate’s Technical Advisory Board (TAB). The TAB consists of a group of senior technology leaders within Redgate (Lead Software Engineers (LSEs), Head of IT, Head of Product Engineering) and is led by the CTO. LSEs take responsibility for reviewing proposed changes within a week.
+
+Comments from all other interested parties are more than welcomed.
 
 ## How is the radar built?
 
-We use [Thoughtworks Tech Radar](https://radar.thoughtworks.com/) to generate our Tech Radar. The radar is backed by a single CSV file (that should nicely [render](https://help.github.com/articles/rendering-csv-and-tsv-data/)). CSV files are parsed using `d3.js` so please see their [documentation](https://d3-wiki.readthedocs.io/zh_CN/master/CSV/) for escaping rules. All changes to the Tech Radar should be completed via a PR and merged by someone else.
```

Comment:
This change may need reverted - I think ReadTheDocs still use `master`.

Reply:
Good spot, thx! How about linking to  `latest` instead of `master`? https://d3-wiki.readthedocs.io/zh_CN/latest/CSV/ ?

Reply:
I'm not sure on their distinction between `master` and `latest`, but I can't immediately see a difference and doubt it'd cause any problems.

Linking to `latest` sounds like a good idea to me 👍 Nice one.